### DISCUSSION TITLE
IOMMU (Intel VT-d) is enabled but Pre-boot DMA protection is disabled

### DISCUSSION THREAD

**Discussion by sjuk** [fwupd/fwupd#7981] [Q&A]
**Describe the question**
The following test failed and I wonder if there is anything I can do to improve the situation.
I followed the links from (--show-all) and searched the internet, but I didn't found any information on how to solve the issue.
The latest BIOS version is already installed and I didn't found any matching setting in the BIOS.

```shell
[..]
✘ Pre-boot DMA protection:       Disabled: https://fwupd.github.io/libfwupdplugin/hsi.html#org.fwupd.hsi.PrebootDma
[..]
```

**fwupd version information**

```shell
fwupdmgr --version
compile   com.hughsie.libxmlb           0.3.18
compile   com.hughsie.libjcat           0.2.0
compile   org.freedesktop.fwupd         1.9.24
runtime   org.freedesktop.fwupd-efi     1.4
compile   org.freedesktop.gusb          0.4.8
runtime   com.hughsie.libxmlb           0.3.x
runtime   com.hughsie.libjcat           0.2.0
runtime   org.freedesktop.gusb          0.4.8
runtime   org.kernel                    6.8.0-47-generic
runtime   org.freedesktop.fwupd         1.9.24
```

**Comment by superm1** [comment]
You should reach out to the manufacturer for advise.  Not all manufacturers export a setting for this, and furthermore some of them will turn it on implicitly when other security settings are set. 

There's nothing that fwupd can do here, it's just reporting.

**Comment by sjuk** [comment]
I've asked the manufacturer of the motherboard for help.

**Comment by sjuk** [comment]
Answer from the manufacturer of the motherboard (Gigabyte): "We regret to inform you that due to the age of the motherboard, our team is no longer able to develop or maintain new BIOS versions. We ask for your understanding."
